src/routes/api/users/UserController/UserController.js

```diff
@@ -0,0 +1,30 @@
+import db from '../../../../models/index';
+/**
+ *
+ *
+ * @class Users
+ */
+class Users {
+  /**
+   *
+   *
+   * @static
+   * @param {obj} req
+   * @param {obj} res
+   * @returns {obj} res
+   * @memberof Users
+   */
+  static async getAllUsers(req, res) {
+    const result = await db.user
+      .findAll({
+        attributes: ['username', 'firstname', 'lastname', 'email', 'updatedAt', 'createdAt', 'bio', 'image']
+      });
+    delete result.password;
```

Comment:
Attention to details? Why is line 22 still here. Mutation?

Reply:
Thanks for the feedback. I have removed that.